index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<link rel="stylesheet" type="text/css" href="style.css">
+		<title>Colin's Website</title>
+	</head>
+	<body>
+		<nav>
+			<a class="navbar" href="#info">About Me</a>
+			<a class="navbar" href="#portfolio">Portfolio</a>
+			<a class="navbar" href="#contact">Contact</a>
+		</nav>
+				<div class="about-me">
+					<h1>Welcome to My Page!</h1>
+					<section class="me">	
+						<img class="personal-photo" src="IMG_0368.jpg">
+						<p class="intro">Greetings, my name is Colin Meier and this webpage will be used to showcase the work I have completed throughout my time at iO Academy.</p>
+					</section>
+					<section id="info">	
+						<h2>About Me</h2>
+						<p class="interests">I was born in New York City and have spent much of my career up until now following my passion in wildlife conservation and eco-tourism. My favorite animal is the clouded leopard, pictured below, and I hope to one day travel to Southeast Asia and try to see one in the wild. In addition to my love for animals and ecology I am also a rabid team sports fan, supporting most New York based clubs in addition to Borussia Dortmund for European football.</p>
+					</section>
+					<section class="photos">
```

Comment:
the class name "photos" makes sense when you read the HTML and see that it's inside of the about-me div, however class names need to make sense in isolation from all the other class names. so it would be more advisable to name it e.g. about-me-photos, because then you can tell it is related to the about-me div. better yet, i'd call it about-me-photos-container, because a name such as "photos" might make someone think the element is a photo.